USE-CASES/transportation.md

```diff
@@ -0,0 +1,88 @@
+## Title: <Pick a descriptive title>
+
+### Submitter(s):
+
+Zoltan Kis
+
+### Reviewer(s):
+
+Jennifer Lin, Michael McCool
+
+### Tracker Issue ID:
+
+<please leave blank>
+
+### Category:
+
+<please leave blank>
+
+### Class:
+
+<please leave blank>
+
+### Status:
+
+<please leave blank>
+
+### Target Users
+
+A Smart City managing roads, transportation tracking and control systems, route information systems, commuting and public transport, vehicles, on-demand transportation, self driving fleets, vehicle information and control systems, infrastructure sharing and payment system, smart parking, smart vehicle servicing, emergency monitoring, etc.
+
+Shipping, air cargo, train cargo transportation systems.
+
+### Motivation:
+
+Smart cities manage several systems for controlling transportation, public transport and commuting, autonomous and human driven vehicles.
+Thing Description templates could be defined in many subdomains to help integration or interworking between multiple systems.
+
+Transportation of goods can be optimized at global level by enhancing interoperability between vertical systems.
+
+### Expected Devices:
+
+Vehicle (human driven, self-driving, isolated or part of fleet).
+Connected vehicles (cars, ships, airplanes, trains, buses etc)
+Airport and port management.
+Public transport vehicles (train, metro, tram, bus, minibus), mobility as a service (ride sharing, bicycle sharing, scooters etc).
+Transportation network planning and management (hubs, backbones, sub-networks, last mile network).
+Electronic timetable management system.
+Road information system (routes, conditions, navigation).
+Road control system (e.g. virtual rails).
+Traffic management services, e.g. intelligent traffic light system with localization and identification (by satellite, radio frequency identification, cameras etc).
+Emergency monitoring and data/location sharing.
+
+### Expected Data:
+
+Vehicle data (identification, location, speed, route, selected vehicle data).
+Weather and climate data.
+
+### Dependencies:
+
+Localization technologies.
+Automotive data.
+Cloud integration.
+
+### Description:
+
+Transportation system implementers will be able to use a unified data description model accoss various systems.
+
+#### Variants:
+
+There will be different verticals, such as:
+
+* Smart City public transport
+* Smart City traffic management
+* Smart city vehicle management
+* Cargo traffic management
+* Cargo vehicle management
```

Comment:
Cargo is ship, air, road, rail and last mile
Stakeholders are very different companies, however shared identifiers for packages, source and target location are desirable, shipping options, like dangerous goods, priority